### ISSUE TITLE
Easier access to mobile dev locally

### ISSUE THREAD

**Issue by jwoertink** [luckyframework/lucky_cli#602]
Granted, it's easy these days to open dev tools and select responsive design to test CSS, but we all know that 90% of the time when you load your site up on an actual mobile device, the browsers don't behave the same.

To make this easier on people, when you run `lucky dev` locally, we should print a giant QR code to the terminal that you can point your phone camera at to load up your site locally. This would just generate a local URL like `http://192.123.1.1:3000` or whatever. The mobile dev framework Expo has something like this, except theirs does `expo://`.

**Comment by jwoertink** [comment]
@kevinsjoberg I was just thinking about this and the proc manager runner deal.. If we ever decided to do this, we'd have to make sure that we can print multiple lines to the console without breaks in between from other processes. I just wanted to write that down while I was thinking about it.

**Comment by kevinsjoberg** [comment]
@jwoertink good point, I don't think that should be a problem. We have full control over the IO, so could just buffer whatever comes in until we've drawn the QR code. There are however, other technical challenges to overcome. What happens when the process redraws the screen (for instance, if they have a progress bar)? Given I'm going down the PTY route for the purpose of making sure the processes output are handled correctly (e.g., no clipping, colors shown correctly and such), we would need to experiment with this.
